examples/kafka/cluster.yaml

```diff
@@ -2,21 +2,26 @@
 apiVersion: kafka.aws.crossplane.io/v1alpha1
 kind: Cluster
 metadata:
-  name: kafka-test-server
+  name: example
 spec:
   forProvider:
+    region: us-east-1
     brokerNodeGroupInfo:
-      clientSubnets:
-        - subnet-f2ccb898
-        - subnet-aac361d6
+      clientSubnetRefs:
+        - name: sample-subnet1
+        - name: sample-subnet2
+      securityGroupRefs:
+        - name: example
```

Comment:
I think we can use `sample-cluster-sg` here

Reply:
yes - i will change it with rebase